.github/tests/upstream-authority-cert-manager/values.yaml

```diff
@@ -0,0 +1,7 @@
+spire-server:
+  upstreamAuthority:
+    disk:
+      enabled: false
```

Comment:
```suggestion
```

This is the default already, so can be removed.

Reply:
oh. Didn't notice that. thanks.